src/parsers/ZillowParser.js

```diff
@@ -27,14 +27,27 @@ export default class ZillowParser {
   }
 
   get rent() {
-    const [rent] = $('.ds-home-details-chip').text().match(priceRegExp()) || [null]
-    return extractNumber(rent)
+    const container = this.document.querySelector('.data-column-container .summary-container')
```

Comment:
I do not like jQuery's `$` but I think it would be better not to mix `document.query...` and `$`

Reply:
I improved this part of code with latest commit. Please, take a look